release notes/v0.43.0.md

```diff
@@ -0,0 +1,124 @@
+k6 v0.43.0 is here! :tada:
+
+Notable changes in this release include:
+- [xk6-browser](https://github.com/grafana/xk6-browser) is now bundled in k6 as an experimental module, and usable without a separate binary or compilation step!
+- Native support for JavaScript's `async`/`await`.
+- A new experimental JavaScript module for distributed tracing.
+- Large refactoring of core components to simplify the code base, improve maintainability, and increase test coverage.
+- Bug fixes, UX improvements, and maintenance.
+
+Keep reading for the details.
+
+## Breaking changes
+
+- [#2807](https://github.com/grafana/k6/pull/2807) Use non-zero exit codes for tests aborted by Ctrl+C or the REST API.
+
+  Aborting a test run with Ctrl+C will now exit with code `105`, and stopping via the REST API will exit with code `103`.
+
+- [#2903](https://github.com/grafana/k6/pull/2903) JS modules are now cached by absolute, and not relative paths.
+
+  This fixes a regression introduced in v0.42.0 ([#2773](https://github.com/grafana/k6/pull/2773)), but has the side-effect of changing how modules are cached internally, which may cause issues for scripts that relied on the previous behavior.
```

Comment:
@mstoykov Do you want to expand here on what exactly users should look out for with this breaking change? I'm not sure I summarized it properly.

Reply:
OK, I was just going by the `breaking change` label in #2903 :smile:

Should we move this to the "Fixes" section instead?

And I think your suggestion goes into too much detail about why this was a problem. Can you instead update the description of #2903 with that information, and we can have a one sentence summary here? I'll try to summarize what you mention.

Reply:
I will resolve this as I update #2903 and made another edit suggestion https://github.com/grafana/k6/pull/2908#discussion_r1102770660